ui/lib/open-api-explorer/addon/components/swagger-ui.js

```diff
@@ -49,16 +52,41 @@ export default class SwaggerUiComponent extends Component {
     };
   }
 
+  // the operationId values in the spec are dasherized
+  // camelize the values so they match the function names in the generated API client SDK
+  CamelizeOperationIdPlugin() {
+    return {
+      wrapComponents: {
+        operation:
+          (Original, { React }) =>
+          (props) => {
+            const { operation } = props;
+            const operationId = operation.get('operationId');
```

Comment:
Just my curiosity, why does this use `.get` ?

Reply:
This whole [plugin system](https://swagger.io/docs/open-source-tools/swagger-ui/customization/plugin-api/) was honestly pretty confusing to figure out but `operation` here is an [Immutable Map](https://immutable-js.com/docs/v3.8.2/Map/).